Octokit/Models/Request/NewMerge.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class NewMerge
+    {
+        /// <summary>
+        /// Create a new commit which has multiple parents (i.e. a merge commit)
+        /// </summary>
+        /// <param name="base">The name of the base branch that the head will be merged into</param>
+        /// <param name="head">The head to merge. This can be a branch name or a commit SHA1.</param>
+        /// <param name="commitMessage">Commit message to use for the merge commit. If omitted, a default message will be used.</param>
+        public NewMerge(string @base, string head, string commitMessage)
+        {
+            Base = @base;
+            Head = head;
```

Comment:
The pattern we use for request objects like this is that the `ctor` should only accept values that are absolutely required. Thus we should ensure that `@base` and `head` are not null nor empty. `commitMessage` is optional so it shouldn't be a parameter to the ctor.